fossid-webapp-client/src/main/kotlin/api/identification/markedAsIdentified/File.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2020 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.fossid.api.identification.markedAsIdentified
+
+class File {
```

Comment:
Why is this not a `data class`?

Reply:
What's the difference to `fossid-webapp-client/src/main/kotlin/api/identification/markedAsIdentified/MarkedAsIdentifiedFile.kt`? This certainly deserves class documentation to explain.

Reply:
Why not a data class ?

because this works only with Kotlin/Jackson
In fossId module we use plain Jackson to leverage the `@JsonUnwrapped` annotation which is not supported by Kotlin/Jackson.

What's the difference [...] ?

One is contained in other. I.e. :
```
@JsonUnwrapped(prefix = "file_")
    var file: File? = null
```
in `MarkedAsIdentifiedFile`.

As for the documentation, I explained in another answer that I don't have access to FossID data model and we agree to document theses classes later.